routers/web/repo/issue.go

```diff
@@ -1510,6 +1510,7 @@ func ViewIssue(ctx *context.Context) {
 						canDelete = true
 						ctx.Data["DeleteBranchLink"] = issue.Link() + "/cleanup"
 					}
+					ctx.Data["CanWriteToHeadRepo"] = true
```

Comment:
There is an if statement before this which confirms if you can write to the head repo (only if this data is true the checkbox to change it will be shown)

Reply:
Okay, LGTM then